### PR TITLE
Added drawer close button and fixed drawer inconsistencies

### LINKED ISSUE #741: Application Drawer is missing close button
Keyboard users and mobile users need a button to close the drawer. 

There was one in the design: https://projects.invisionapp.com/d/main?origin=v7#/console/19928472/417192802/preview?scrollOffset=459

### PR DESCRIPTION
**Pull request by gsidebo**
#### What are the relevant tickets?
Closes #741 

#### What's this PR do?
- Adds a close button to the top of our drawer contents
- Fixes styling inconsistencies in our drawer contents (padding, button styling, etc.)

#### How should this be manually tested?
Open drawers from the application dashboard, check them in multiple browser widths, use the close button

#### Screenshots (if appropriate)
ss 2020-06-24 at 13 47 49
ss 2020-06-24 at 13 48 01
ss 2020-06-24 at 13 48 10
ss 2020-06-24 at 13 48 30
ss 2020-06-24 at 13 48 37
ss 2020-06-24 at 13 58 07
ss 2020-06-24 at 13 49 43
ss 2020-06-24 at 13 50 39
ss 2020-06-24 at 14 02 11

### CODE REVIEW

**Comment by gsidebo** [comment]
@abdulkdawson Screenshots ^^^

File: ecommerce/admin.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 class LineAdmin(admin.ModelAdmin):
     """Admin for Line"""
-
+    static / js / components / ResumeLinkedIn_test.js
```
Comment (annagav) [inline ecommerce/admin.py:13]: please remove this line

File: static/scss/drawer.scss [hunk 2]
```diff
@@ -7,27 +7,48 @@
...
+    right: 0;
     max-width: 600px;
-    padding: 25px;
   }
 }
 
 .drawer-wrapper {
   @include media-breakpoint-down(sm) {
-    padding: 20px;
+    padding: 15px 20px 20px 20px;
```
Comment (annagav) [inline static/scss/drawer.scss:17]: This could just be 
```suggestion
    padding: 15px 20px 20px;
```

File: static/scss/drawer.scss [hunk 3]
```diff
@@ -7,27 +7,48 @@
...
   }
 
   @include media-breakpoint-down(xs) {
-    padding: 15px 0;
+    padding: 15px;
   }
 
-  padding: 40px;
+  padding: 15px 35px 35px 35px;
```
Comment (annagav) [inline static/scss/drawer.scss:24]: You can omit the last 35 here

**Comment by annagav** [comment]
I am getting a grey border around the X:
Screen Shot 2020-06-24 at 11 56 46 PM

File: ecommerce/admin.py (hunk 1 above)
Comment (gsidebo) [reply ecommerce/admin.py:13]: 😬

**Comment by gsidebo** [comment]
@annagav Ready for review again. I changed that focus outline to grey

**Comment by abdulkdawson** [comment]
@gsidebo this is looking good.

**Pull request merged**